package/kernel/gpio-button-hotplug/src/gpio-button-hotplug.c

```diff
@@ -100,6 +100,7 @@ static struct bh_map button_map[] = {
 	BH_MAP(KEY_LIGHTS_TOGGLE,	"lights_toggle"),
 	BH_MAP(KEY_PHONE,		"phone"),
 	BH_MAP(KEY_POWER,		"power"),
+	BH_MAP(KEY_POWER2,		"reboot"),
```

Comment:
This change doesn't correspond to the commit description (base-files package). It should go into separate commit.